contracts/contracts/core/BidderRegistry.sol

```diff
@@ -143,10 +139,7 @@ contract BidderRegistry is
         uint256 len = windows.length;
         for (uint256 i = 0; i < len; ++i) {
             uint256 window = windows[i];
-            require(
-                window < currentWindow,
-                "withdraw after window settled"
-            );
+            require(window < currentWindow, WithdrawAfterWindowSettled());
```

Comment:
Again, here I think we should print both the windows. If we dont know the values, we wont be able to debug what is going wrong here.